db/dbConfig.js

```diff
@@ -0,0 +1,4 @@
+const knex = require("knex");
+const knexConfig = require("../knexfile");
+
+module.exports = knex(knexConfig.developement);
```

Comment:
Check out your export.  The word development is misspelled. this might causing an error when you start your server

Reply:
OMG THANK YOU! I was up until 1 am trying to figure out what was wrong